lib/bot.js

```diff
@@ -300,6 +300,22 @@ module.exports = (function() {
 			});
 		},
 
+		getPagesTranscludedBy: function (template, callback) {
```

Comment:
What about ```getPagesTranscluding()```? The thing is that it is not necessary a template that is transcluded...